dhall-csv/csv-to-dhall/Main.hs

```diff
@@ -0,0 +1,6 @@
+module Main where
+
+import qualified Dhall.CSVToDhall
+
+main :: IO ()
+main = putStrLn Dhall.CSVToDhall.hello
```

Comment:
AFAIK this is the more common way to handle abbreviations in CamelCase – I think it's more readable too.

```suggestion
import qualified Dhall.CsvToDhall

main :: IO ()
main = putStrLn Dhall.CsvToDhall.hello
```

Reply:
I made it that way because it's the format that dhall-json package uses. But I agree with you in that it is more readable this way and I'm seeing that dhall-yaml package uses this format for abreviations, so I'll update it. Thanks!

Reply:
Yeah. Maybe we should migrate to this style in `dhall-json` too.